scripts/Tests.py

```diff
@@ -0,0 +1,68 @@
+"""
+Utility functions for running tests and logging the results.
```

Comment:
why not to reuse ``unittest`` module. Something like was done here:
https://github.com/cloudflare/tls-tris/blob/master/_dev/interop_test_runner